hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/routes.json

```diff
@@ -24,8 +24,5 @@
   "/namespace/quota?path=*": "/quota",
   "/task/status": "/taskStatus",
   "/containers/missing": "/missingContainers",
```

Comment:
Please remove this endpoint, as it is not used anymore.

Reply:
In UI Overview.tsx we are using this endpoint.

Reply:
![image](https://user-images.githubusercontent.com/112169209/199978237-8d944e77-f193-4cdd-9009-dba87d44d6b8.png)

Reply:
If we use the `/containers/missing` endpoint there why do we need to specify the `"/missingContainers"` route here? What would happen in the `overview.tsx` if we remove this from the `routes.json`?
```suggestion
```

Reply:
It is  key value pair  /missingContainer is associated  name for route /conatiner/missing we have to mention route in route.json. If we remove from overview.tsx code will break and UI will not work.